src/rattler_build_conda_compat/jinja.py

```diff
@@ -0,0 +1,47 @@
+from __future__ import annotations
+
+from typing import Any
+
+import jinja2
+import yaml
+
+from rattler_build_conda_compat.loader import load_yaml
+
+
+def jinja_env() -> jinja2.Environment:
+    """
+    Create a `rattler-build` specific Jinja2 environment with modified syntax.
+    """
+    return jinja2.Environment(
+        variable_start_string="${{",
+        variable_end_string="}}",
+        trim_blocks=True,
+        lstrip_blocks=True,
+        autoescape=True,
+    )
+
+
+def render_context(context: dict[str, str], jinja_env: jinja2.Environment) -> dict[str, str]:
+    # Process each key-value pair in the dictionary
+    for key, value in context.items():
+        # If the value is a string, render it as a template
+        if isinstance(value, str):
+            template = jinja_env.from_string(value)
+            rendered_value = template.render(context)
+            context[key] = rendered_value
+
+    return context
+
+
+def eval_recipe(recipe_content: dict[str, Any]) -> dict[str, Any]:
```

Comment:
Can you add a doc comment? Its not entirely clear what this function does from just the name and arguments.

Reply:
done